Services/Dashboard/classes/class.ilDashboardGUI.php

```diff
@@ -147,33 +153,21 @@ public function executeCommand(): void
                 break;
 
             case "ilcolumngui":
+                if (strtolower($cmdClass = $this->ctrl->getCmdClass()) === strtolower(ilSelectedItemsBlockGUI::class)) {
+                    $gui = new $cmdClass();
+                    $ret = $this->ctrl->forwardCommand($gui);
+                    if ($ret != "") {
```

Comment:
Please use either `!$ret` or `$ret !== ""` depending on the fact that you know the type of `$ret` or not.